backend/conduit/tags/views.py

```diff
@@ -136,4 +136,17 @@ def review_article(slug, articleSlug):
         if article.is_allTagReviewed():
             article.set_needsReview(False)
     article.save()
-    return article
\ No newline at end of file
+    return article
+
+
+#Route to return an article filtered by tag names
+@blueprint.route('/api/user/tags/articles', methods=('GET',))
+@jwt_optional
+@marshal_with(articles_schema)
+def get_articles_tags(isPublished=None, tag=None, author=None, favorited=None, limit=5, offset=0):
+    tagLists = current_user.profile.followed_tags
+    ans = []
+    if tagLists is not None:
```

Comment:
If the tagLists is none, we should just return the first 5 tags from all the tags